delo_diplomskega_seminarja/fmfdelo.cls

```diff
@@ -67,22 +62,6 @@
   \algrenewtext{For}[3]{\algorithmicfor\ #1 $\gets$ #2\ \algorithmicto\ #3\ \algorithmicdo}
   \algdef{S}[FOR]{ForEach}[2]{\algorithmicforeach\ #1\ \algorithmicin\ #2\ \algorithmicdo}
 }
-\newcommand{\zapisiMetaPodatke}{
-  \hypersetup{pdfencoding=auto, psdextra, pdflang=sl, % ali 'en' za anglesicno
-    bookmarksopen, bookmarksdepth=3,
-    pdfauthor=\@avtor,
-    pdftitle=\@naslov,
-    pdfproducer={pdfTeX},
-    pdfsubject={matematika},
-    pdfkeywords=\@kljucnebesede}
-  % Paket hyperref definira ukaz \C, ki ga zelo zelo verjetno nihce ne bo rabil,
-  % zato ga odstranimo, da lahko \C uporabljamo za kompleksna stevila. Odstraniti
-  % ga moramo sele tu, saj \hypersetup ocitno makro ponovno definira
-  \let\C\undefined
```

Comment:
Ali je kak razlog da se to odstrani? Meni se zdi lepše, če to ostane v predlogi, da je manj razmišljanja o definicijah v (potencialno že obstoječih) nalogah.

Reply:
Razlog je v tem, da ne bi po nepotrebnem smetil razreda. Sploh pa, kot gledam, v zadnjih verzijah `\C` ni več definiran.